src/components/table/use-cases/generations.tsx

```diff
@@ -213,6 +218,27 @@ export default function GenerationsTable({ projectId }: GenerationsTableProps) {
       enableHiding: true,
       enableSorting: true,
     },
+    {
+      accessorKey: "endTime",
+      id: "endTime",
+      header: "End Time",
+      enableHiding: true,
+      enableSorting: true,
```

Comment:
is sorting already supported here?

Reply:
Yes, we are already sorting the startTime. I assumed we only need to enableSorting. It also works locally for me. Is there some magic in the background happing or do I need to do more than just setting the flag here?